Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -138,6 +146,10 @@
     sprite: Clothing/OuterClothing/WinterCoats/coatgen.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/WinterCoats/coatgen.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Acidic: 0.9

```

Comment:
why does geneticist coat have more acid resistance than chem and **cmo** coats?

Reply:
The geneticist winter coat has less resistance than the chem and CMO winter coats. That's what testing shows and what the tooltips say. The 0.75 of the chem and CMO coats are multiplied by the incoming damage, reducing the damage by 25%. The geneticist winter coat is 0.90 which is only a 10% reduction.